src/app/lib/config/toggles/liveConfig.js

```diff
@@ -28,6 +28,9 @@ export default {
     value:
       '(afrique|arabic|hausa|indonesia|korean|pashto|persian|somali|swahili|urdu)',
   },
+  onDemandRadioSchedule: {
+    enabled: true,
```

Comment:
Should we leave this `false` on live for now until we've tested everything?

Reply:
I guess we could but I'm not using toggles at all

Reply:
What will stop the schedules going straight to live?

Reply:
> What will stop the schedules going straight to live?

Ah ok I didn't realise this was a pain to test locally. So we want to get this on test but not live? OK I'll make the changes to do that.

Reply:
Yep :) Gives Lily and Jack some time to give it a look over. Shouldn't be anything wrong since it's identical data to Live Radio but good for peace of mind.